3.4/security-security-options.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+description: arangod provides a variety of options to make a setup more secure
+---
+# Server security options
+
+_arangod_ provides a variety of options to make a setup more secure. 
```

Comment:
"variety" for 2 options seems a little exaggerated. We shouldn't this paragraph as-is from 3.5 with a whole lot more options.